### DISCUSSION TITLE
Version 2.3.10 broke my implemetation

### DISCUSSION THREAD

**Discussion by SanibelJack** [Security-Onion-Solutions/securityonion#2395] [Unsupported Versions]
Since I did a soup update on the master the master has services that are not running and the forward node is all but dead. HELP.

**Comment by TOoSmOotH** [comment]
How can anyone help you if you post absolutely no details?

**Reply by SanibelJack** [reply]
I did an a post from yesterday but no one responded

**Reply by SanibelJack** [reply]
On the forward node, the only service that is running is Docker and filebeat, everything else is in error state. On the master elastalert is in error, eleasticsearch is in error, flleet is in error, mysql is in error, and playbook is in error state.  AllI did was run soup on the Master...

**Reply by TOoSmOotH** [reply]
on the manager do a sudo so-docker-refresh and then a sudo salt-call state.highstate queue=True then check so-status

**Reply by SanibelJack** [reply]
So the log file says that I'm out of drive space...  How do I free up space on Centos?

**Reply by SanibelJack** [reply]
When I do the so-docker-refresh I get 500 Internal server error on several services, mysql, steno, and thehive. Is that on your end or mine?

**Reply by weslambert** [reply]
The so-docker-refresh should work fine.  I would check to ensure you can resolve those URLs manually.  As far as space is concerned, you could clear data out of /nsm, or you may check to see what's taking up the most space on `/` with something like `sudo du -hs /*`.

**Comment by SanibelJack** [comment]
The quay.io URL’s?

Jack Voth
CISSP, CISM, CEH
Sanibel Cyber Technology, LLC
Jack.voth@sanibelcybertech.com<mailto:Jack.voth@sanibelcybertech.com>
www.sanibelcybertechnology.com<http://www.sanibelcybertechnology.com>
239-249-9311
